### ISSUE TITLE
Does not detect npm or yarn

### ISSUE THREAD

**Issue by moltar** [camacho/install-deps-postmerge#14]
I have a  project with `package.json` and `package-lock.json`, when I run:

```sh
npx install-deps-postmerge
```

It runs:

```
❯ npx install-deps-postmerge --help
yarn install v1.22.4
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
```

**Comment by camacho** [comment]
Yes, this is currently the case. The published version of the package is looking for the wrong file. I have a fix that is on master, but the code is not compiling correctly so I haven't been able to publish a new working version.

**Comment by camacho** [comment]
This has been fixed in v2.0.1
